app/assets/javascripts/govuk_publishing_components/components/layout-super-navigation-header.js

```diff
@@ -134,8 +134,8 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     var $navMenuLinks = this.$navMenu.querySelectorAll('li a')
     var $firstNavLink = $navMenuLinks[0]
     var $lastNavLink = $navMenuLinks[$navMenuLinks.length - 1]
-    var $searchMenuLinks = this.$searchMenu.querySelectorAll('li a')
-    var $lastSearchLink = $searchMenuLinks[$searchMenuLinks.length - 1]
+    var $searchMenuTabable = this.$searchMenu.querySelectorAll('li a, input, button')
```

Comment:
Small nitpick, should "Tabable" be "Tabbable" used in the variable names?

Reply:
Yeah, fair nitpick! I hovered between the two a bit. Happy to go with "bb"!